torch/nn/modules/activation.py

```diff
@@ -432,9 +432,10 @@ def extra_repr(self) -> str:
         return inplace_str
 
 class Hardswish(Module):
-    r"""Applies the hardswish function, element-wise, as described in the paper:
+    r"""Applies the Hardswish function, element-wise, as described in the paper:
+    `Searching for MobileNetV3 <https://arxiv.org/abs/1905.02244>`__.
```

Comment:
I think this should end in a single underscore, but I haven't seen the rendered docs yet

Reply:
you are right, I recompiled with "_" and still renders into a link (not surprisingly, as the docs say the same :)) )